src/core/server/capabilities/integration_tests/capabilities_service.test.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { BehaviorSubject } from 'rxjs';
+import { ByteSizeValue } from '@kbn/config-schema';
+import supertest from 'supertest';
+
+import { HttpService, InternalHttpServiceSetup } from '../../http';
+import { configServiceMock } from '../../config/config_service.mock';
+import { contextServiceMock } from '../../context/context_service.mock';
+import { loggingServiceMock } from '../../logging/logging_service.mock';
+import { Env } from '../../config';
+import { getEnvOptions } from '../../config/__mocks__/env';
+import { CapabilitiesService, CapabilitiesSetup } from '..';
+
+const coreId = Symbol('core');
+const env = Env.createDefault(getEnvOptions());
+
+const createHttpServer = (): HttpService => {
```

Comment:
we alreay have the same setup for integration tests in `http` service. probably we should extract this helper to a separate file to simplify testing. WDYT? can be done in a separate PR

Reply:
You're right. Will avoid some boilerplate in future core route. Will be done in current PR.